test/integration/http-service.yaml.tmpl

```diff
@@ -43,8 +43,6 @@ spec:
           - "{{.port1}}"
           - --port
           - "{{.port2}}"
-          - --version
```

Comment:
default value for version is "". i don't see any reference to "unversioned" anywhere in the tests.